go/test_multipart_force.sh

```
@@ -0,0 +1,96 @@
+# プロジェクトディレクトリ
+PROJECT_DIR="/Users/forsteri/work/Projects/s3-sdk-experiments"
+GO_DIR="$PROJECT_DIR/go"
+TEST_DATA_DIR="$PROJECT_DIR/test-data"
+
+# バケット名
+BUCKET="s3-experiment-bucket-250615"
+
+echo -e "${BLUE}=== Multipart Upload Function Test ===${NC}"
+echo ""
+
+cd "$GO_DIR"
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)